xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp

```diff
@@ -900,6 +902,9 @@ int CVideoPlayerVideo::OutputPicture(const DVDVideoPicture* src, double pts)
     return EOS_DROPPED;
   }
 
+  m_processInfo.SetVideoDimensions(pPicture->iWidth, pPicture->iHeight);
```

Comment:
I think DAR is more useful for displaying (e.g. DAR:1.77).
PAR can be computed fom DAR and width/height if needed.
So, either is fine (but need to make sure all codecs are consistent).